maestrowf/conductor.py

```diff
@@ -48,8 +48,7 @@
 LOGGER = logging.getLogger(__name__)
 
 # Formatting of logger.
-LFORMAT = "%(asctime)s - %(name)s:%(funcName)s:%(lineno)s - " \
-               "%(levelname)s - %(message)s"
+LFORMAT = "[%(asctime)s: %(levelname)s] %(message)s"
```

Comment:
This change in format is a good call, however it's less verbose which makes debugging potentially harder. Might be worth considering adding a condition to the logger utility to use a different format when the logger level is above a certain value. What do you think @ben-bay?

Reply:
Agreed, especially for DEBUG. I played around with this a little yesterday, but couldn't quite get it working which is why it's not included. Will try again

Reply:
This morning, I looked more for information on changing formatting based on log level, but it doesn't seem to be a very common question people have-- so either it's super obvious, or no one is doing it...

Reply:
Interesting that that question hasn't come up. I was also a little unclear, apologies. I was advocating for something like the following before passing the LFORMAT to :

```
# Globabls
DBG_LFORMAT = "%(asctime)s - %(name)s:%(funcName)s:%(lineno)s - " \
                             "%(levelname)s - %(message)s"
LFORMAT = "[%(asctime)s: %(levelname)s] %(message)s"

....
# Where we set up the file handler and stream handler
LFORMAT = 
if log_level == logging.DEBUG:
    lformat = DBG_LFORMAT
else:
    lformat = LFORMAT

LOG_UTIL.add_file_handler(log_path, lformat, args.debug_lvl)
```

Something to this effect was what I was thinking -- since the person is wanting debug, it's beneficial to be explicit, so explicitly force it to be outside the log util.

Reply:
I see, that's a lot easier. So all the message levels, even info, would be in the debug format.

Reply:
This works! Awesome!